lib/src/modal/modal-header.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input } from '@angular/core';
+import { ActiveModal } from './active-modal';
+
+@Component({
+    selector: 'hc-modal-header',
+    template: `<div class="hc-modal-header">
+                   <ng-content></ng-content>
+                   <button *ngIf="showCloseButton" class="hc-modal-close-button" (click)="dismissModal()">&times;</button>
+               </div>`
+})
+export class ModalHeaderComponent {
+
+    @Input() showCloseButton: boolean = true;
```

Comment:
I know we had a close button up in the header of some modals back in older prototypes I did - but I've stopped including that.  I think it's confusing to have two methods for dismissing a modal; best if the user always knows they have their save/cancel buttons at the bottom, and they have to pick one of those.  So I'd recommend we remove the showCloseButton option from this component.

Reply:
The close button and its options have been removed from ModalHeaderComponent